types/http-assert/http-assert-tests.ts

```diff
@@ -1,26 +1,113 @@
 import httpAssert = require('http-assert');
-import { HttpError } from 'http-errors';
+
+declare function unknown(): unknown;
+
+// assert()
```

Comment:
Is this a leftover comment? Maybe it's a description of the test set, not sure. Maybe it's worth splitting the tests into two separate files as we did in `koa` package? This file becomes pretty long now...